### PR TITLE
post scores

### PR DESCRIPTION
**Pull request by ZSuraj**

### CODE REVIEW

File: functions/intern.js [hunk 1]
```diff
@@ -222,4 +222,45 @@ app.post("/getIntern", async (req, res) => {
   }
 });
 
+app.post("/postScores", async (req, res) => {
+  // if (req.header("skillrazr-sub-app") !== env.INTERN_API_HEADER_KEY_VALUE) {
```
Comment (tukuna30) [inline functions/intern.js:226]: Remove the comments, as it's required.

File: functions/intern.js [hunk 2]
```diff
@@ -222,4 +222,45 @@ app.post("/getIntern", async (req, res) => {
...
+    )}_${dateObj.getFullYear()}`;
+
+    const internRef = db.collection("interns").doc(email);
+    const internDoc = await internRef.get();
+    const performanceData = internDoc.data().performanceData;
+
+    if (!performanceData[monthYear]) {
+      performanceData[monthYear] = {};
+    }
+    const Scores = {
```
Comment (tukuna30) [inline functions/intern.js:247]: Scores => scores

**Review by tukuna30 — CHANGES_REQUESTED** [review]

File: functions/intern.js (hunk 1 above)
Comment (ZSuraj) [reply functions/intern.js]: fixed it

File: functions/intern.js (hunk 2 above)
Comment (ZSuraj) [reply functions/intern.js]: renamed to scoresObj if i use scores will conflict with scores destructured variable

File: functions/intern.js [hunk 3]
```diff
@@ -222,4 +222,45 @@ app.post("/getIntern", async (req, res) => {
...
+      code_reviews: scores.codeReview,
+      develpoment: scores.development,
+      learning: scores.learning,
+      testing: scores.testing,
+    };
+
+    if (!performanceData[monthYear].scores) {
```
[the remark this answers is not in the document]
Comment (ZSuraj) [reply functions/intern.js:254]: OK sure

**Pull request merged**